core/storage/collection/gae_models.py

```diff
@@ -219,9 +222,10 @@ class CollectionRightsSnapshotContentModel(
 
     @staticmethod
     def get_export_policy():
-        """This model's export_data function has not been implemented yet.
-        However, it is pending implementation per #8523.
-        """
+        """This model's export_data function implementation is still pending.
+
+       TODO(#8523): Implement this function.
+       """
```

Comment:
I can't reply to the original conversation thread on GitHub (there is no input field). But this comment, though marked resolved, is not resolved: https://github.com/oppia/oppia/pull/8210#discussion_r376107939

Please let's do two things: (a) stop resolving comments (just reply to them), and (b) make sure to actually do the code changes for stuff that you've marked "done" or equivalent. This is a sensitive PR and we need to have a good way of keeping track of the requests and changes, otherwise the reviewer will need to review the entire PR every single time.

Reply:
Sorry Sean, I'm a bit confused. How is this not resolved? The comment you linked to is for the CollectionModel and the snippet of code you have highlighted is a docstring that I have changed to match the docstring you have recommended. Not sure what went wrong here... Please let me know!

Reply:
We talked about this in conversation -- I think you are pointing to the wrong model here.

Reply:
Done.